### ISSUE TITLE
Update focus state on admin tool bar in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12549]
## Description
In the CMS, the admin toolbar has no visible focus indicator on the elements within the toolbar. This is an accessibility error and should be remediated.

## Updates to be made
The focus state of the elements in this admin tool bar should mimic the hover state with the additional of a focus outline
- Text and Icon color should be `var(--va-blue-dark)`. Currently it is white
- Add standard green focus outline:
  - outline: var(--focus-outline);
    box-shadow: var(--focus-box-shadow);

### Screenshots
Current hover state:
![image](https://user-images.githubusercontent.com/106678594/217943899-34215d70-feeb-4a20-af1d-a8fe6c3b4b92.png)

Current Focus state:
![image](https://user-images.githubusercontent.com/106678594/217943850-7e797175-9210-4ff8-8f56-2dafae651f97.png)

Green focus outline:
![image](https://user-images.githubusercontent.com/106678594/217943980-f847c0a4-d050-45f0-a5cd-fadb03ac3c01.png)

## Acceptance Criteria
- [ ] Update text focus color
- [ ] Update icon focus color
- [ ] Add focus outline
- [ ] Accessibility lead review

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by EWashb** [comment]
@edmund-dunn @laflannery do you think this is still on target for this sprint?

**Comment by laflannery** [comment]
@EWashb I spoke with Edmund and this had to be deprioritized because he was working on the email notifications.  I apologize I didn't let you know sooner - I had thought it was stretch because of the [Sprint plan](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12598) so I just didn't realize

**Comment by EWashb** [comment]
@laflannery you're right and I'm wrong. Idk why I put the sprint label on that. I shouldn't have. We didn't present it in the sprint plan as a commitment. It clearly says stretch. I'll remove the sprint label (that I shouldn't have put there to begin with)
